build.gradle.kts

```diff
@@ -25,6 +25,6 @@ plugins {
     id("com.google.devtools.ksp") version "1.9.20-1.0.14" apply false
 }
 
-tasks.register("clean", Delete::class) {
-    delete(rootProject.buildDir)
+tasks.register<Delete>("clean") {
```

Comment:
@C10udburst can you just check this pr for anything that might fuck the build process